org.lflang/src/lib/core/federated/RTI/rti.c

```diff
@@ -689,7 +693,28 @@ bool send_tag_advance_if_appropriate(federate_t* fed) {
 }
 
 /**
- * Handle a logical tag complete (LTC) message.
+ * For all federates transitively downstream of the specified federate, determine
```

Comment:
Also this sentence works without the word "transitive".  I would just describe this as a recursive function that visits downstream federates and sends them a TAG if it's safe for them to advance.

Reply:
For the comment, I agree that recursive is more accurate.